fasthtml/core.py

```diff
@@ -441,15 +441,15 @@ async def _wrap_call(f, req, params):
     "multi-swap": "https://unpkg.com/htmx-ext-multi-swap@2.0.0/multi-swap.js", 
     "path-deps": "https://unpkg.com/htmx-ext-path-deps@2.0.0/path-deps.js", 
     "remove-me": "https://unpkg.com/htmx-ext-remove-me@2.0.0/remove-me.js",
-    "ws": "https://unpkg.com/htmx-ext-ws/ws.js",
-    "chunked-transfer": "https://unpkg.com/htmx-ext-transfer-encoding-chunked/transfer-encoding-chunked.js"
+    "ws": "https://unpkg.com/htmx-ext-ws@2.0.1/ws.js",
+    "chunked-transfer": "https://unpkg.com/htmx-ext-transfer-encoding-chunked@0.4.0/transfer-encoding-chunked.js"
 }
 
 # %% ../nbs/api/00_core.ipynb
-htmxsrc   = Script(src="https://unpkg.com/htmx.org@next/dist/htmx.min.js")
-fhjsscr   = Script(src="https://cdn.jsdelivr.net/gh/answerdotai/fasthtml-js@1.0.4/fasthtml.js")
-surrsrc   = Script(src="https://cdn.jsdelivr.net/gh/answerdotai/surreal@main/surreal.js")
-scopesrc  = Script(src="https://cdn.jsdelivr.net/gh/gnat/css-scope-inline@main/script.js")
+htmxsrc   = Script(src="https://unpkg.com/htmx.org@2.0.3/dist/htmx.min.js")
+fhjsscr   = Script(src="https://cdn.jsdelivr.net/gh/answerdotai/fasthtml-js@1.0.4/fasthtml.min.js")
```

Comment:
IMO we should used minified scripts for these 3 really small ones. I don't think it's much of a win in terms of perf, and it's harder to debug and understand.

Reply:
Yeh thats very true, will reset them back since they're tiny scripts